app/components/ProtectedRoute/index.js

```diff
@@ -1,51 +1,51 @@
-/**
- *
- * ProtectedRoute
- *
- */
+// /**
+//  *
+//  * ProtectedRoute
+//  *
+//  */
 
-import React from 'react';
-import { Redirect, Route } from 'react-router-dom';
-import PropTypes from 'prop-types';
-import routeConstants from '@utils/routeConstants';
+// import React from 'react';
+// import { Redirect, Route } from 'react-router-dom';
+// import PropTypes from 'prop-types';
+// import routeConstants from '@utils/routeConstants';
```

Comment:
Remove all the commented code if you don't have to use. When you think you will need it, you need to add them then otherwise you shouldnt have commented code except for those that will explain about the code.